plugin.rb

```diff
@@ -15,28 +15,25 @@
 
 after_initialize do
   module ::DiscourseEncrypt
-    PLUGIN_NAME          = 'discourse-encrypt'
-
-    PUBLIC_CUSTOM_FIELD  = 'encrypt_public'
-    PRIVATE_CUSTOM_FIELD = 'encrypt_private'
-    TITLE_CUSTOM_FIELD   = 'encrypted_title'
-
-    Store = PluginStore.new(PLUGIN_NAME)
+    PLUGIN_NAME = 'discourse-encrypt'
 
     def self.set_key(topic_id, user_id, key)
```

Comment:
Nitpick: I am wondering if we can get rid of all these methods: `set_key`, `get_key` and `del_key`. I originally introduced them because I did now want to interact with the plugin store directly (i.e. always pass plugin name or an instance of `PluginStore`, build key, etc).